libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.stories.ts

```diff
@@ -47,15 +47,15 @@ storiesOf('Arrow Variation', module)
       template: `
       <div style="${containerStyle}">
         <div style="${arrowDivStyle}">
-          <gor-arrow-variation size='sm' [value]="-1"></gor-arrow-variation>
+          <gor-arrow-variation size='sm' [value]="-1" [pdfPrint]="true"></gor-arrow-variation>
           Small
         </div>
         <div style="${arrowDivStyle}">
-          <gor-arrow-variation size='md' [value]="-1"></gor-arrow-variation>
+          <gor-arrow-variation size='md' [value]="-1" [pdfPrint]="true"></gor-arrow-variation>
           Medium
         </div>
         <div style="${arrowDivStyle}">
-          <gor-arrow-variation size='lg' [value]="-1"></gor-arrow-variation>
+          <gor-arrow-variation size='lg' [value]="-1" [pdfPrint]="true"></gor-arrow-variation>
```

Comment:
Stories won't be printed, so you don't need to change all of them to true. So if you want to illustrate that the png files are working correctly, you can choose only one case in each `add`, set print to `true`, and leave the others with the default value.